packages/framework/src/Model/Order/PromoCode/PromoCode.php

```diff
@@ -22,15 +29,51 @@ class PromoCode
 
     /**
      * @var string
-     * @ORM\Column(type="text", unique=true)
+     * @ORM\Column(type="text", unique=false)
```

Comment:
false is a default value, I believe it's not necessary to be explicit about it

Reply:
oh, good catch, fixed in 4cab29f422